### PR TITLE
feat(inputs.dns_query): Add IP field(s)

### LINKED ISSUE #10452: dns_query input plugin - Pull FQDN IPs when querying
### Proposal
Pull FQDN IPs when querying, FAO @srebhan 

### Current behavior
Doesn't do this :)

### Desired behavior
When you query a FQDN it currently pulls all the nice info through which is great, but it would be nice to show what IP address(es) the lookup found as well like an nslookup.

### Use case
The idea is to them push all this into Grafana.

Thanks

### PR DESCRIPTION
**Pull request by srebhan**
- [x] Updated associated README.md.
- [ ] Wrote appropriate unit tests.
- [x] Pull request title or commits are in [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0/#summary)

resolves #10452

This PR adds a new `include_fields` option which, if containing `"IP"` or `"all IPs"`, will allow to output the IP address or all IP addresses for the queried domain as field(s). Please note that the IPs are not ordered in any way but are output in the order they arrive in the DNS response.

### CODE REVIEW

File: plugins/inputs/dns_query/README.md [hunk 1]
```diff
@@ -33,8 +33,14 @@ See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
...
 
-  ## Query timeout in seconds.
-  # timeout = 2
+  ## Query timeout
+  # timeout = "2s"
+
+  ## Include the specified additional properties in the resulting metric.
+  ## The following values are supported:
+  ##    IP        -- return IP of the first A and AAAA answer
+  ##    "all IPs" -- return IP of the all A and AAAA answers with index
```
Comment (powersj) [inline plugins/inputs/dns_query/README.md:42]: It feels weird to see an option with mixed case + space and both should really be quoted, what do you think about:

```suggestion
  ##    "first_ip"    -- return IP of the first A and AAAA answer
  ##    "all_ips"     -- return IPs of the all A and AAAA answers with index
```

**Review by powersj — COMMENTED** [review]
Looks great as usual, only comment is on the naming of the new include field options

**Review by powersj — APPROVED** [review]

**Comment by powersj** [comment]
Thanks - you or I can merge once tests pass!

**Comment by Hipska** [comment]
Do you have an example of such a metric?

**Pull request merged**